.compatibility_tests/compatibility_test_1_15/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "compatibility_test_1_15"
+version = "1.16.0"
+authors = ["HQS Quantum Simulations <info@quantumsimulations.de>"]
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.70"
+categories = ["science", "simulation"]
+homepage = "https://github.com/HQSquantumsimulations/qoqo"
+repository = "https://github.com/HQSquantumsimulations/qoqo"
+documentation = "https://docs.rs/roqoqo/"
+readme = "../README.md"
+description = "Compatibility tests for roqoqo"
+include = ["src*", "build.rs", "LICENSE", "README.md"]
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[workspace]
+
+[dependencies]
+test_roqoqo_1_15 = { package = "roqoqo", version = "=1.15.2" }
+test_roqoqo_derive_1_15 = { package = "roqoqo-derive", version = "=1.15.2" }
```

Comment:
Is there a particular reason why we use 1.15.2 instead of 1.15.0?

Reply:
Looking at the other compatibility tests, we have always pinned everything to the latest patch version. I guess it's simply for the sake of testing the most up-to-date versions.